php/classes/controllers/integrations/elementor/main.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace SeriouslySimplePodcasting\Controllers\Integrations\Elementor;
+
+use SeriouslySimplePodcasting\Controllers\Integrations\Elementor\Widgets\Elementor_Episode_List_Widget;
+use SeriouslySimplePodcasting\Controllers\Integrations\Elementor\Widgets\Elementor_Html_Player_Widget;
+use SeriouslySimplePodcasting\Controllers\Integrations\Elementor\Widgets\Elementor_Media_Player_Widget;
+use SeriouslySimplePodcasting\Controllers\Integrations\Elementor\Widgets\Elementor_Subscribe_Links;
+
+final class Main {
```

Comment:
Ok, so I know that the Elementor documentation uses this singleton pattern, but I'm not a fan of it. So, can we do the following

- [ ] rename the class file to class-elementor-widgets.php
- [ ] rename the class to `class Elementor_Widgets'
- [ ] remove the $_instance property and the instance method

That way it's just a regular class and we can instantiate it by doing

`$elementor_widgets = new Elementor_Widgets();`

In the main plugin file.

Reply:
Yep, makes total sense. Done